wrappers/services.go

```diff
@@ -876,9 +903,9 @@ func EnsureSnapServices(snaps map[*snap.Info]*SnapServiceOptions, opts *EnsureSn
 	return context.reloadModified()
 }
 
-// AddSnapServicesOptions is a struct for controlling the generated service
+// generateSnapServicesOptions is a struct for controlling the generated service
 // definition for a snap service.
-type AddSnapServicesOptions struct {
+type generateSnapServicesOptions struct {
```

Comment:
the comment for QuotaGroup here need to change a bit too I think

Reply:
Good point! Will update it!